No configuration file found.
No pyproject.toml file found.
stubPath /mnt/data/2020-11-API-misuse/Repair_API_Misuse/api-misuse-repair/2_parse_dl_repos/repo_download_v2/misuse_repos/EleutherAI_gpt-neo/26aef0a236e7053ab3cf9127962b1d50d6a6dc4c/typings is not a valid directory.
Assuming Python platform Linux
Searching for source files
Found 1 source file
/home/apr/data/2020-11-API-misuse/Repair_API_Misuse/api-misuse-repair/2_parse_dl_repos/repo_download_v2/misuse_repos/EleutherAI_gpt-neo/26aef0a236e7053ab3cf9127962b1d50d6a6dc4c/a/data/create_tfrecords.py
  /home/apr/data/2020-11-API-misuse/Repair_API_Misuse/api-misuse-repair/2_parse_dl_repos/repo_download_v2/misuse_repos/EleutherAI_gpt-neo/26aef0a236e7053ab3cf9127962b1d50d6a6dc4c/a/data/create_tfrecords.py:9:8 - error: Import "ftfy" could not be resolved (reportMissingImports)
  /home/apr/data/2020-11-API-misuse/Repair_API_Misuse/api-misuse-repair/2_parse_dl_repos/repo_download_v2/misuse_repos/EleutherAI_gpt-neo/26aef0a236e7053ab3cf9127962b1d50d6a6dc4c/a/data/create_tfrecords.py:12:6 - error: Import "lm_dataformat" could not be resolved (reportMissingImports)
  /home/apr/data/2020-11-API-misuse/Repair_API_Misuse/api-misuse-repair/2_parse_dl_repos/repo_download_v2/misuse_repos/EleutherAI_gpt-neo/26aef0a236e7053ab3cf9127962b1d50d6a6dc4c/a/data/create_tfrecords.py:13:6 - error: Import "tokenizers" could not be resolved (reportMissingImports)
  /home/apr/data/2020-11-API-misuse/Repair_API_Misuse/api-misuse-repair/2_parse_dl_repos/repo_download_v2/misuse_repos/EleutherAI_gpt-neo/26aef0a236e7053ab3cf9127962b1d50d6a6dc4c/a/data/create_tfrecords.py:14:6 - error: Import "transformers" could not be resolved (reportMissingImports)
  /home/apr/data/2020-11-API-misuse/Repair_API_Misuse/api-misuse-repair/2_parse_dl_repos/repo_download_v2/misuse_repos/EleutherAI_gpt-neo/26aef0a236e7053ab3cf9127962b1d50d6a6dc4c/a/data/create_tfrecords.py:15:6 - error: Import "tqdm" could not be resolved (reportMissingImports)
  /home/apr/data/2020-11-API-misuse/Repair_API_Misuse/api-misuse-repair/2_parse_dl_repos/repo_download_v2/misuse_repos/EleutherAI_gpt-neo/26aef0a236e7053ab3cf9127962b1d50d6a6dc4c/a/data/create_tfrecords.py:16:6 - error: Import "encoders" could not be resolved (reportMissingImports)
  /home/apr/data/2020-11-API-misuse/Repair_API_Misuse/api-misuse-repair/2_parse_dl_repos/repo_download_v2/misuse_repos/EleutherAI_gpt-neo/26aef0a236e7053ab3cf9127962b1d50d6a6dc4c/a/data/create_tfrecords.py:142:33 - error: Operator ">" not supported for "None" (reportOptionalOperand)
  /home/apr/data/2020-11-API-misuse/Repair_API_Misuse/api-misuse-repair/2_parse_dl_repos/repo_download_v2/misuse_repos/EleutherAI_gpt-neo/26aef0a236e7053ab3cf9127962b1d50d6a6dc4c/a/data/create_tfrecords.py:144:16 - error: Operator "<" not supported for types "int" and "Unknown | None"
    Operator "<" not supported for types "int" and "None" (reportGeneralTypeIssues)
  /home/apr/data/2020-11-API-misuse/Repair_API_Misuse/api-misuse-repair/2_parse_dl_repos/repo_download_v2/misuse_repos/EleutherAI_gpt-neo/26aef0a236e7053ab3cf9127962b1d50d6a6dc4c/a/data/create_tfrecords.py:172:28 - error: Operator ">" not supported for "None" (reportOptionalOperand)
  /home/apr/data/2020-11-API-misuse/Repair_API_Misuse/api-misuse-repair/2_parse_dl_repos/repo_download_v2/misuse_repos/EleutherAI_gpt-neo/26aef0a236e7053ab3cf9127962b1d50d6a6dc4c/a/data/create_tfrecords.py:184:16 - error: Operator "<" not supported for types "int" and "Unknown | None"
    Operator "<" not supported for types "int" and "None" (reportGeneralTypeIssues)
10 errors, 0 warnings, 0 infos 
Completed in 1.698sec

time cost(in seconds): 1.8684895038604736
